bot/cogs/clickup.py

```diff
@@ -0,0 +1,152 @@
+# coding=utf-8
+from aiohttp import ClientSession
+
+from discord import Colour, Embed
+from discord.ext.commands import AutoShardedBot, Context, command
+
+from bot.constants import ADMIN_ROLE, CLICKUP_KEY, CLICKUP_SPACE, CLICKUP_TEAM, DEVOPS_ROLE, MODERATOR_ROLE, OWNER_ROLE
+from bot.decorators import with_role
+
+CREATE_TASK_URL = "https://api.clickup.com/api/v1/list/{list_id}/task"
+GET_TASKS_URL = "https://api.clickup.com/api/v1/team/{team_id}/task"
+PROJECTS_URL = "https://api.clickup.com/api/v1/space/{space_id}/project"
+
+# Don't ask me why the below line is a syntax error, but that's what flake8 thinks...
+SPACES_URL = "https://api.clickup.com/api/v1/team/{team_id}/space"  # flake8: noqa
+TEAM_URL = "https://api.clickup.com/api/v1/team/{team_id}"
+
+HEADERS = {
+    "Authorization": CLICKUP_KEY,
+    "Content-Type": "application/json"
+}
+
+
+class ClickUp:
+    """
+    ClickUp management commands
+    """
+
+    # Set statuses: Open, In Progress, Review, Closed
+    # Open task
+    # Assign task
+
+    def __init__(self, bot: AutoShardedBot):
+        self.bot = bot
+
+    @command(name="clickup.tasks()", aliases=["clickup.tasks", "tasks"])
+    @with_role(MODERATOR_ROLE, ADMIN_ROLE, OWNER_ROLE, DEVOPS_ROLE)
+    async def tasks(self, ctx: Context, list: int = None, status: str = None):
+        """
+        Get a list of tasks, optionally on a specific list or with a specific status
+        """
+
+        params = {}
+
+        if list:
```

Comment:
naw, man. don't do it. you'll break my heart.

Reply:
I'm wondering what else to call this, since it really is a clickup list that you're putting in

Reply:
`task_list`? even `list_` would be better, but I don't think it needs to come to that.

Reply:
I was just thinking about it because this will appear as a command argument in the help as well

Reply:
`task_list` still seems fairly inoffensive to me. Isn't that what it is?

Reply:
Hm, yeah, I suppose you're right about that